VS/src/msgpack.light.benchmark/BeerConverter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+using MsgPack.Light;
+
+namespace msgpack.light.benchmark
+{
+    internal class BeerConverter : IMsgPackConverter<Beer>
+    {
+        private IMsgPackConverter<string> _stringConverter;
+
+        private IMsgPackConverter<List<string>> _listStringConverter;
+
+        private IMsgPackConverter<float> _floatConverter;
+        private MsgPackContext _context;
+
+        public void Write(Beer value, IMsgPackWriter writer)
+        {
+            if (value == null)
+            {
+                _context.NullConverter.Write(null, writer);
+                return;
+            }
+
+            writer.WriteArrayHeader(4);
```

Comment:
Default msgpack-cli behaviour is serialization to array. We just repeating after them.